File: 1440-convert-integer-to-the-sum-of-two-no-zero-integers/convert-integer-to-the-sum-of-two-no-zero-integers.py
class Solution:
    def getNoZeroIntegers(self, n: int) -> List[int]:
        num1 = 1
        num2 = n - 1
        while True:
            if '0' in str(num2) or '0' in str(num1):
                num1 +=1
                num2 -=1
            else:
                break
        return [num1,num2]

        
        
        

File: 1440-convert-integer-to-the-sum-of-two-no-zero-integers/convert-integer-to-the-sum-of-two-no-zero-integers.js
var getNoZeroIntegers = function(n) {
    for (let i= 0;i<n;i++){
        if (String(i).includes('0'))
        continue;
    j = n - i;
    if (String(j).includes('0'))
    continue;
    return [i,j];
    }
};